### PR TITLE
Returned flow checks to CI

### PR DESCRIPTION
**Pull request by bestander**
Test Plan:
- see if flow checks don't crash Circle

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @ide and @mkonicek to be potential reviewers.

File: circle.yml [hunk 1]
```diff
@@ -47,11 +47,10 @@ test:
     - source scripts/circle-ci-android-setup.sh && waitForAVD
 
   override:
-    # TODO flow is disabled see t11343811 and 0.25.1
     # eslint bot
-    # - cat <(echo eslint; npm run lint --silent -- --format=json; echo flow; npm run flow --silent -- check --json) | GITHUB_TOKEN="af6ef0d15709bc91d""06a6217a5a826a226fb57b7" CI_USER=$CIRCLE_PROJECT_USERNAME CI_REPO=$CIRCLE_PROJECT_REPONAME PULL_REQUEST_NUMBER=$CIRCLE_PR_NUMBER node bots/code-analysis-bot.js
+    - cat <(echo eslint; npm run lint --silent -- --format=json; echo flow; npm run flow --silent -- check --json) | GITHUB_TOKEN="af6ef0d15709bc91d""06a6217a5a826a226fb57b7" CI_USER=$CIRCLE_PROJECT_USERNAME CI_REPO=$CIRCLE_PROJECT_REPONAME PULL_REQUEST_NUMBER=$CIRCLE_PR_NUMBER node bots/code-analysis-bot.js
     # JS tests for dependencies installed with npm3
-    # - npm run flow check
+    - npm run flow check
```
Comment (avaly) [inline circle.yml]: `npm run flow -- check` if you expect this to run `flow check`

**Comment by avaly** [comment]
Might wanna trigger a few rebuilds on Circle CI to be sure it works consistently. LGTM otherwise!

**Comment by facebook-github-bot-0** [comment]
@bestander updated the pull request.

**Comment by bestander** [comment]
@avaly done

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/932082960271122/int_phab) to review.

**Pull request closed without merging**